README.md

```diff
@@ -2,3 +2,51 @@
 
 # CHORDS MQTT Orchestrator
 
+This package provides a tool to push MQTT messages to a [CHORDS portal]().
+
+## Setup
+
+### Clone this repository
+
+```shell
+git clone ...
+```
+
+### Modify `config.yaml`
```

Comment:
We should probably clarify that this modification needs to happen after the `installation` step. For users who are going to clone the repo, they may end up making changes to the wrong file `chords_mqtt/config.yaml` from the source code.